SlipWeatherUITests/SlipWeatherUITests.swift

```diff
@@ -2,7 +2,7 @@
 //  SlipWeatherUITests.swift
 //  SlipWeatherUITests
 //
-//  Created by Etienne Le Louet on 26/05/2019.
+//  Created by Etienne Le Louët on 26/05/2019.
```

Comment:
Petite faute ici attention

```suggestion
//  Created by Etienne L'alouette on 26/05/2019.
```

Reply:
Ah oui, effectivement, merci beaucoup pour cette correction, je l'avais laissé passé celle-là ><"
Heureusement que "Le Louët" ne s'est pas retrouvé dans la branche master, on aurait du faire un push -force ou même un reset --hard ><"